stubs/setuptools/setuptools/_distutils/command/install_data.pyi

```diff
@@ -0,0 +1,19 @@
+from _typeshed import Incomplete
+
+from ..cmd import Command
+
+class install_data(Command):
```

Comment:
I could mark this as `@deprecated` given there's no planned replacement https://github.com/pypa/setuptools/issues/4070